### DISCUSSION TITLE
How to add luckysheet build into angular project?

### DISCUSSION THREAD

**Discussion by ProfessionProgrammer** [dream-num/Luckysheet#996] [Q&A]
I build the lucky sheet project, and I am trying to include it in the angular-CLI project. Please anyone can help me with this.

Thanx.

**Comment by WodenWang820118** [comment]
I've tried and built one for Angular 13.
1. Use luckyexcel to import.
2. Use the luckysheet to render.
3. See `angular.json` for `styles` and `scripts`.
4. The implementation is in the `app/components/panel`.
5. A useful reference: https://github.com/mengshukeji/Luckyexcel/blob/master/src/index.html 
https://github.com/WodenWang820118/angular-csv

**Reply by JCyxMio** [reply]
Thankyou,I have this Question too ,it is worked now by your help!

**Comment by nicoesiea** [comment]
May I ask for more explanation, please? For example:
- How to import luckysheet in my ts file?
- How to import the styles file?
For the CSS files, I did it like this:
`styleUrls: [
    '../../../../node_modules/luckysheet/dist/css/luckysheet.css',
    '../../../../node_modules/luckysheet/dist/plugins/css/pluginsCss.css',
    '../../../../node_modules/luckysheet/dist/plugins/plugins.css',
    '../../../../node_modules/luckysheet/dist/assets/iconfont/iconfont.css'

  ],`

Is there a better way how to do it?
